cfpb/cfgov-refresh
# Updates ESLint config

Updates ESLint to latest.

## Changes

- Updates `grunt-eslint` to `10.0.0`.
- Replaces and updates deprecated rules.
- Changes `wrap-iife` to `inside` to conform with organization
standards.
- Fixes code comment grammar.

## Testing

- Run `npm install` to pull in latest.
- Run `grunt lintjs` and see that there are 543 problems, but ESLint not being up-to-date isn't one.

## Review

- @sebworks 
- @jimmynotjim
